src/main/java/com/k3ntako/HTTPServer/Request.java

```diff
@@ -90,6 +90,10 @@ public void setParams(HashMap<String, String> params) {
     return this.params;
   }
 
+  public String getParam(String key) {
+    return this.params.get(key);
+  }
```

Comment:
look at other examples, see if there is a commonly used name

Reply:
It seems that many places call it route parameters, so I renamed `params` to `routeParams`

Examples:
https://expressjs.com/en/guide/routing.html
https://laravel.com/docs/5.0/routing#route-parameters
https://guides.rubyonrails.org/routing.html